spec.bs

```diff
@@ -512,6 +517,11 @@ The <dfn for=Navigator method>runAdAuction(|config|)</dfn> method steps are:
 1. Let |auctionConfig| be the result of running [=validate and convert auction ad config=] with
   |config| and true.
 1. If |auctionConfig| is failure, then [=exception/throw=] a {{TypeError}}.
+1. Optionally, [=exception/throw=] a "{{NotAllowedError}}" {{DOMException}}.

```

Comment:
Would https://wicg.github.io/turtledove/#validate-and-convert-auction-ad-config be a better place to check this, maybe in step 3 since we're doing validation anyways? Semantically it seems nice, but I don't have very strong opinion.

Reply:
That sounds good, since that will invoke the checks in a central place. Moving it.